instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientCommonAttributesGetter.java

```diff
@@ -8,7 +8,7 @@
 import javax.annotation.Nullable;
 
 /** An interface for getting attributes common to database clients. */
-public interface DbClientCommonAttributesGetter<REQUEST> {
+public interface DbClientCommonAttributesGetter<REQUEST, RESPONSE> {
```

Comment:
Adding a type parameter is a breaking change. @trask is this fine or do you have suggestions?

Reply:
I think it's ok